torch/jit/_shape_functions.py

```diff
@@ -439,6 +439,34 @@ def embedding(
     size.append(weight[1])
     return size
 
+def embedding_bag_padding_idx(weight: List[int], indices: List[int], offsets: List[int], scale_grad_by_freq: bool, mode: int, sparse: int, per_sample_weights: Optional[List[int]], include_last_offset: bool, padding_idx: Optional[int]) -> Tuple[List[int], List[int], List[int], List[int]]):
```

Comment:
You do not need to include the arguments that are not used inside the shape function.

Reply:
makes sense.